src/AnonymousUser/Base64CookieEncoder.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace InsightArchitectures.AnonymousUser
+{
+    /// <summary>
+    /// Default cookie value encoder/decoder. Uses base64 for serialisation.
+    /// </summary>
+    public class Base64CookieEncoder : ICookieEncoder
+    {
+        /// <summary>
+        /// Deserialises a base64 value into clear text.
+        /// <param name="encodedValue">A base64 value. Returns null if argument is null</param>
+        /// </summary>
```

Comment:
```
        /// <summary>
        /// Deserialises a base64 value into clear text.
        /// <param name="encodedValue">A base64 value to deserialize.</param>
        /// <returns>Returns null if <paramref name="encodedValue" /> is null.</returns>
        /// </summary>
```

You can use the `<returns />` section and the `<paramref />` tag. See: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags